internal/project/auto/golang.go

```diff
@@ -156,19 +156,6 @@ func (builder *builder) processDirectory(path string) error {
 
 	rootPath := filepath.Join(builder.rootPath, dir)
 
-	if builder.meta.VersionPackage == "" {
```

Comment:
I'm not sure why this got dropped, as this is not backwards compatible, and will require all projects to define a config object for it?

Reply:
Returned as it was.